wa/framework/configuration/plugin_cache.py

```diff
@@ -79,6 +79,11 @@ def add_configs(self, plugin_name, values, source):
             msg = "Source '{}' has not been added to the plugin cache."
             raise RuntimeError(msg.format(source))
 
+        if caseless_string(plugin_name) == 'global':
```

Comment:
Should be `if caseless_string(plugin_name) in ['global', 'config']:`

Reply:
Fixed